frontend/src/metabase/actions/containers/ActionCreator/utils.ts

```diff
@@ -164,7 +165,7 @@ export const convertQuestionToAction = (
     cleanQuestion.parameters(),
   );
   const visualization_settings = removeOrphanSettings(
-    addMissingSettings(formSettings, parameters),
+    syncFieldsWithParameters(formSettings, parameters),
     parameters,
   );
```

Comment:
It seems like `removeOrhpanSettings` is no longer relevant here. AFAIK it was supposed to remove field settings that don't have a backing parameter anymore 🤔 I'm afraid I could break it while trying to encapsulate this action/question workaround 😢

Reply:
Great catch! It seems not needed anymore, removed